cdap-ai-ext-vertexai/src/test/java/io/cdap/cdap/ai/ext/vertexai/VertexAIConfigurationTest.java

```diff
@@ -0,0 +1,144 @@
+package io.cdap.cdap.ai.ext.vertexai;
+
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.Assert;
+import org.junit.Test;
+
+/**
+ * Unit tests for the {@link VertexAIConfiguration} class.
+ *
+ * This class contains tests to verify the behavior of the {@link VertexAIConfiguration} class
+ * when initialized with various sets of properties.
+ */
+public class VertexAIConfigurationTest {
+
+  /**
+   * Tests the {@link VertexAIConfiguration} class when all required properties are provided.
+   *
+   * Verifies that the configuration object correctly initializes and retrieves all properties:
+   * project ID, location, model name, and prompt.
+   */
+  @Test
+  public void testVertexAIConfigurationWithAllProperties() {
+
+    Map<String, String> properties = new HashMap<>();
+    properties.put(VertexAIConfiguration.PROJECT_ID, "test-project-id");
```

Comment:
Extract the frequently used strings into constants and re-use it throughout the tests
Eg - 
`private static final String PROJECT_ID = "test-project-id";`

Reply:
Done